tests/util/verify.sh

```diff
@@ -280,30 +281,59 @@ __cluster_state() {
     # TEMP WORKAROUND, don't check istio-system
     kubectl get ns -o name | sed '/istio-system/d'
     kubectl get all --ignore-not-found -n default
-    kubectl get istiooperators --ignore-not-found -n default 2>&1 | grep -v "error: the server doesn't have a resource type"
-    kubectl get destinationrules --ignore-not-found -n default -n istio-system 2>&1 | grep -v "error: the server doesn't have a resource type"
-    kubectl get envoyfilters --ignore-not-found -n default -n istio-system 2>&1 | grep -v "error: the server doesn't have a resource type"
-    kubectl get gateways --ignore-not-found -n default -n istio-system  2>&1| grep -v "error: the server doesn't have a resource type"
-    kubectl get serviceentries --ignore-not-found -n default -n istio-system 2>&1 | grep -v "error: the server doesn't have a resource type"
-    kubectl get sidecars --ignore-not-found -n default -n istio-system 2>&1 | grep -v "error: the server doesn't have a resource type"
-    kubectl get virtualservices --ignore-not-found -n default -n istio-system 2>&1 | grep -v "error: the server doesn't have a resource type"
-    kubectl get workloadentries --ignore-not-found -n default -n istio-system 2>&1 | grep -v "error: the server doesn't have a resource type"
-    kubectl get authorizationpolicies --ignore-not-found -n default -n istio-system 2>&1 | grep -v "error: the server doesn't have a resource type"
-    kubectl get peerauthentications --ignore-not-found -n default -n istio-system 2>&1 | grep -v "error: the server doesn't have a resource type"
-    kubectl get requestauthentications --ignore-not-found -n default -n istio-system 2>&1 | grep -v "error: the server doesn't have a resource type"
+    kubectl get istiooperators --ignore-not-found -n default 2>&1
```

Comment:
I'm wondering why the grep is no longer needed. It was a workaround (hack actually) because some of the `profile_none` tests were failing because the Istio CRDs didn't exist when the snapshot was taken (which output these missing type messages), but they were there at the end of the test.

Reply:
it was causing failures when I was running. And CI seems fine as well.